modules/solid_mechanics/src/userobjects/CrackFrontDefinition.C

```diff
@@ -1046,75 +1054,73 @@ CrackFrontDefinition::rotateToCrackFrontCoords(const ColumnMajorMatrix tensor, c
 void
 CrackFrontDefinition::calculateRThetaToCrackFront(const Point qp, const unsigned int point_index, Real & r, Real & theta) const
 {
-  unsigned int num_nodes(_ordered_crack_front_nodes.size());
+  unsigned int num_points(_crack_front_points.size());
   Point p = qp;
-  Point closest_node(0.0);
-  RealVectorValue closest_node_to_p;
+  Point closest_point(0.0);
+  RealVectorValue closest_point_to_p;
 
   const Point *crack_front_point = getCrackFrontPoint(point_index);
-  RealVectorValue crack_front_node_rot = rotateToCrackFrontCoords(*crack_front_point,point_index);
+  RealVectorValue crack_front_point_rot = rotateToCrackFrontCoords(*crack_front_point,point_index);
 
   RealVectorValue crack_front_edge = rotateToCrackFrontCoords(_tangent_directions[point_index],point_index);
 
   Point p_rot = rotateToCrackFrontCoords(p,point_index);
-  p_rot = p_rot - crack_front_node_rot;
+  p_rot = p_rot - crack_front_point_rot;
 
   if (_treat_as_2d)
   {
     //In 2D, the closest node is the crack tip node and the position of the crack tip node is (0,0,0) in the crack front coordinate system
     //In case this is a 3D mesh treated as 2D, project point onto same plane as crack front node.
     //Note: In the crack front coordinate system, z is always in the tangent direction to the crack front
-    p_rot(2) = closest_node(2);
-    closest_node_to_p = p_rot;
+    p_rot(2) = closest_point(2);
+    closest_point_to_p = p_rot;
 
     //Find r, the distance between the qp and the crack front
     RealVectorValue r_vec = p_rot;
     r = r_vec.size();
-
   }
   else
   {
     // Loop over crack front nodes to find the one closest to the point qp
     Real min_dist = std::numeric_limits<Real>::max();
-    for (unsigned int nit = 0; nit != num_nodes; ++nit)
+    for (unsigned int nit = 0; nit != num_points; ++nit)
     {
-      Node & crack_front_node = _mesh.node(_ordered_crack_front_nodes[nit]);
-      RealVectorValue crack_node_to_current_node = p - crack_front_node;
-      Real dist = crack_node_to_current_node.size();
+      Point crack_front_point = _crack_front_points[nit];
```

Comment:
minor: opportunity to use reference instead of copy here. It's likely this is optimized out since this is a trivial basic block